starter/script.js

```diff
@@ -1 +1,125 @@
+  } else {
+    console.log("You jam the ham sandwich into your mouth to buy yourself more time to decide. A ridiculous choice, but it works. In a way it works too well: you begin to choke. Luckily a guy wearing 3 fedoras performs the heimlich maneuver on you. You flee home in embarassment. THE END");
+  }
+}
+
+function yoLoAndBehold(){
+  console.log("You get assigned to a team and then starting CRUSHING it. People cheer insanely. You're making every shot, it's like you"+
+  " are assigning ping pong ball variables to nested a nested array of cups and beer linked to an array where each key is a wasted opponent."+
+  " Wow. You needed this. That last thought kind of scared you. Maybe your metaphorical vocabulary could be enriched beyond turnips and coding."+
+  " At any rate, you jump around and have a good time and make some new friends. Whoa, this super wasted dude somehow wearing 3 fedoras"+
+  " approaches you for an epic high-five. Oh wow look at that, he is now airborne.");
+  var whatDoin = prompt("Do you jump and high-five in mid-air? [1]COMMIT TO IT YEAH [2]nah imma dodgggge");
+  if (inputCrusher(whatDoin,2)==1){
+    console.log("You leap up in the air to hit the high-five but his hand misses yours and your heads collide. Fedoras fly everywhere"+
+    " and, as you lose consciousnes everyone's voices drop in pitch just like a movie cliche. That's kind of lame. Hey wait, you feel pretty conscious but"+
+    " everything has gone black. That's weird. You become aware of anxious voices. You are able to gather that you went flying into a window and landed on the "+
+    " network switch, which apparently shocked you, causing you convulse. People think you are dead. In fact, they are saying you have no pulse. Weird."+
+    " You then see yourself lying on the ground. You realize you are watching yourself from a webcam. Seeing your old body. From a web cam."+
+    " Hey wait. You now have a hypothesis: your consciousness was transferred to the GA LAN network. You verify this by running some commands."+
+    " Yup, all the output you are seeing supports this. You discover you are actually now a part of GArnet. You conceal all of your code, the code"+
+    " that is you, so no one will notice, or like make a feature request or raise an issue. You're fine the way you are. 'Yeah that's right,' you think"+
+    " to yourself, 'I am fine the way I am; this is fine. I'll just help new GA students with their code in ways they can't detect, and fix bugs in GArnet."+
+    " It's like I'm a ghost at Hogwarts or something.' THE END");
+  } else {
+    console.log("Good move. That dude was way off balance. He totally tumbles into the table and like knocks a bunch of cups over, spilling beer everywhere."+
+    " Everyone yells things like 'YOLO' and 'party foul' and 'the lizard people control everything from the lower-left 4th dimension.'"+
```

Comment:
I was basically laughing and having my brain reprogrammed the entire time I was reading this... but I lost it at this line...